lerna.json

```diff
@@ -6,5 +6,5 @@
     "lib/octicons_helper",
     "lib/jekyll-octicons"
   ],
-  "version": "independent"
+  "version": "7.3.0"
```

Comment:
Does this get auto updated by lerna or do we have to manually update it?

Reply:
running the `npm run bump` command runs the appropriate lerna command which gives us a nice prompt for deciding.

![image](https://user-images.githubusercontent.com/54012/39831254-c1003d14-5378-11e8-9d9c-f3ead488dfff.png)